From ceaf5f9b28da640e45f9427ff37db3831b65e4c5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sun, 5 Sep 2010 14:23:50 +0000
Subject: [PATCH 12/15] gettextize: git-revert mark the "me" variable for translation
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The "me" variable is used in many later messages to indicate whether
we're using revert or cherry-pick. Mark it for translation for later
use. Also leave a TRANSLATORS message explaining what these are for.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/revert.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/builtin/revert.c b/builtin/revert.c
index ece0c0f..2f319f8 100644
--- a/builtin/revert.c
+++ b/builtin/revert.c
@@ -552,7 +552,9 @@ static int revert_or_cherry_pick(int argc, const char **argv)
 	struct rev_info revs;
 
 	git_config(git_default_config, NULL);
-	me = action == REVERT ? "revert" : "cherry-pick";
+	/* TRANSLATORS: This is used in several error messages indicating
+	   the name of the current program */
+	me = action == REVERT ? N_("revert") : N_("cherry-pick");
 	setenv(GIT_REFLOG_ACTION, me, 0);
 	parse_args(argc, argv);
 
-- 
1.7.2.3.313.gcd15

